app/views/issue_comments/new.html.erb

```diff
@@ -1 +0,0 @@
-<%= render "new_form" %>
```

Comment:
I don't think this file should have been removed. Any reason to explain the deletion?